src/test/lib/TestHelpers.h

```diff
@@ -112,6 +112,49 @@ struct ClearGlobalVersionListScope {
     }
 };
 
+//
+// Global parameter setting might affect other tests' behavior.
+// This sets back the original value
+//
+struct GlobalSettingScope {
+    uint32_t Parameter;
+    uint32_t BufferLength;
+    void* OriginalValue;
+    GlobalSettingScope(uint32_t Parameter) : Parameter(Parameter), BufferLength(0), OriginalValue(nullptr) {
```

Comment:
My preference would be to do it this way:
```suggestion
    uint32_t Parameter;
    uint32_t BufferLength {0};
    void* OriginalValue {nullptr};
    GlobalSettingScope(uint32_t Parameter) : Parameter(Parameter) {
```